binancial/features/trend_and_momentun.py

```
@@ -0,0 +1,94 @@
+def trend_and_momentun(df):
```

Ruff (0.8.2)

Missing return type annotation for public function `trend_and_momentun`

(ANN201)